utility/src/main/java/org/oppia/android/util/parser/TestGlideImageLoader.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.android.util.parser
+
+import android.content.Context
+import android.graphics.drawable.Drawable
+import org.oppia.android.util.caching.AssetRepository
+import org.oppia.android.util.caching.CacheAssetsLocally
+import javax.inject.Inject
+
+/** A [TestGlideImageLoader] that is used in tests. */
+class TestGlideImageLoader @Inject constructor(
+  context: Context,
+  @CacheAssetsLocally private val cacheAssetsLocally: Boolean,
+  assetRepository: AssetRepository
+) : GlideImageLoader(context, cacheAssetsLocally, assetRepository) {
+
+  override fun loadDrawable(
+    imageDrawableResId: Int,
+    target: ImageTarget<Drawable>,
+    transformations: List<ImageTransformation>
+  ) {
+    if (target is ImageViewTarget) {
```

Comment:
Please add a brief explanation for what this is achieving (& why the default glide function is insufficient here.

Reply:
Done.